packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -513,13 +486,13 @@ class _RangeSliderState extends State<RangeSlider> with TickerProviderStateMixin
   // non-zero displacement is negative, then the left thumb is selected, and if its
   // positive, then the right thumb is selected.
   static final RangeThumbSelector _defaultRangeThumbSelector = (
-    TextDirection textDirection,
-    RangeValues values,
-    double tapValue,
-    Size thumbSize,
-    Size trackSize,
-    double dx, // The horizontal delta or displacement of the drag update.
-  ) {
+      TextDirection textDirection,
```

Comment:
Should these be formatted back to the way they were?

Reply:
Done